Tests/test_stdconsole.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         # This is relative to working directory so the path related tests (e.g.'print __name__')
         # return predictable results.
 
-        self.tmpdir = "tmp" #os.path.join(self.temp_dir, "tmp")
+        self.tmpdir = os.path.join(self.temp_dir, "tmp")
```

Comment:
A specific reason for using `self.temp_dir` here?
Using it keeps this test not-parallel safe (unlike `self.temporary_dir`, which is segregated per framework).

Reply:
No specific reason beyond that's what was commented. I'll see if I can get rid of `self.temp_dir`...